### ISSUE TITLE
concourse/pool-resource:1.1.5 (-alpine) is missing ssh-agent

### ISSUE THREAD

**Issue by dsharp-pivotal** [concourse/pool-resource#59]
We use pool-resource with a private git repository and set `source.private_key` with an ssh private key. Today we started getting the below error:

```
/opt/resource/out: line 11: ssh-agent: not found
sh: you need to specify whom to kill
```

Checking the images, ssh-agent does seem to be missing from the 1.1.5-alpine image:
```
→ docker run --rm -it concourse/pool-resource:1.1.4 which ssh-agent
/usr/bin/ssh-agent
→ docker run --rm -it concourse/pool-resource:1.1.5-ubuntu which ssh-agent
/usr/bin/ssh-agent
→ docker run --rm -it concourse/pool-resource:1.1.5-alpine which ssh-agent
(no output)
```

Seems to be caused by #58 

CC: @vito

**Comment by dsharp-pivotal** [comment]
Resources are also turning orange when they run their check:

```
run check step: run check step: check: resource script '/opt/resource/check []' failed: exit status 1

stderr:
/opt/resource/check: line 11: ssh-agent: not found
sh: you need to specify whom to kill
```

**Comment by vito** [comment]
Ah yep, definitely caused by that PR. Thanks for the report - we'll ship patch (and figure out why our tests don't cover this) tomorrow!

**Comment by aoldershaw** [comment]
Shipped 1.1.6 with the patch:

```sh
docker run --rm -it concourse/pool-resource:1.1.6-alpine which ssh-agent
/usr/bin/ssh-agent
```
